ast.go

```diff
@@ -778,25 +779,25 @@ const (
 	LessEqualStr    = "<="
 	GreaterEqualStr = ">="
 	NotEqualStr     = "!="
-	InStr           = "in"
-	NotInStr        = "not in"
-	LikeStr         = "like"
-	NotLikeStr      = "not like"
-	RegexpStr       = "regexp"
-	NotRegexpStr    = "not regexp"
-	MatchStr        = "match"
-	NotMatchStr     = "not match"
-	GlobStr         = "glob"
-	NotGlobStr      = "not glob"
+	InStr           = " in "
```

Comment:
This change is a bit weird but that's how I designed things actually. These constants are used to indicate what kind of operator the `CmpExpr` node of the AST holds. It is weird to have the spaces in there to indicate that. For example, the `IN` operator is indicated by the value `" in "` of the `Operator` field. It's more natural to expect just `"in"` without spaces. The problem there is that the same variable is used for indicating the kind of operator and how that operator will be encoded into a string. This is not your fault actually. Just mentioning this nuance.

Reply:
I was a little unsure of how to handle the spaces in these `const`s. I revisited this when I started looking at how to better remove spaces surrounding parenthesis.  I think I found a good way to leave these strings as they were, and manage parenthesis better.